# An array of poses with a header for global reference.

Header header

recyclerbot/Cylinder[] cylinders
